bevy_lint/src/lints/pedantic/borrowed_reborrowable.rs

```diff
@@ -145,16 +146,9 @@ impl<'tcx> LateLintPass<'tcx> for BorrowedReborrowable {
 
         // A list of argument types, used in the actual lint check.
         let arg_types = fn_sig.inputs().skip_binder();
-        // A list of argument names, used to check and skip `&mut self`.
-        let arg_names = cx.tcx.fn_arg_names(def_id);
```

Comment:
While `fn_arg_names()` was just renamed to `fn_arg_idents()`, I didn't realize it at the time, so I refactored this to use [`Body::params`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_hir/hir/struct.Body.html#structfield.params).